KEDamage.cs

```diff
@@ -52,13 +51,26 @@ private static float CalculateKEAttenuationSimple(float pierce, float distance,
             return finalPierce;
         }
         
-        private static float CalculatePostERAPierce(float pierce, float eraFrictionMultiplier)
+        private static float CalculatePostERAPierce(float pierce, float eraFractionMultiplier)
         {
-            float finalPierce = pierce*(1-eraFrictionMultiplier);
+            float finalPierce = pierce*(1-eraFractionMultiplier);
+            return finalPierce;
+        }
+        #endregion
 
-            // Use some callback function to update target's era value here
```

Comment:
We'd want to update *object here* by gathering data from where the function is called.